frontend/src/component/commandBar/CommandBar.tsx

```diff
@@ -181,6 +202,11 @@ export const CommandBar = () => {
                             icon={'flag'}
                             items={flags}
                         />
+                        <CommandResultGroup
+                            groupName={'Projects'}
+                            icon={'flag'}
```

Comment:
https://github.com/Unleash/unleash/blob/main/frontend/src/component/commandBar/RecentlyVisited/RecentlyVisited.tsx#L53

Here's a way to get the project icon (cellphone coverage bars)